src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
+  padding: 5px;
+  padding: $padding;
+  margin: auto;
+  gap: $gap;
+
+  @each $day, $index in $days {
+    &--start-day--#{$day} {
+      & > .calendar__day:first-child {
+        margin-left: calc(
+          (#{$day-size} * #{$index}) +
+            (#{$gap} * #{$index}) +
+            (#{$padding}) +
+            (#{$border-size}) *
```

Comment:
The variable `$border-size` is used here, but it's not clear if it is defined elsewhere. Ensure this variable is defined to avoid compilation errors.